docs/Running-behind-nginx.md

```diff
@@ -205,7 +205,7 @@ You can also use a unix domain socket. This will also provide a faster route bet
 
 ```
 [web]
-    bind to = unix:/tmp/netdata.sock
+    bind to = unix:/run/netdata/netdata.sock
```

Comment:
Hi @CtrlAltDel64 ,

Please, change the suggestion from `/run` to `/var/run`, because `/run` is a Linux directory only and Netdata also runs on FreeBSD. I am appending a screenshot from FreeBSD to demonstrate this:

![BSD_directory_structure](https://user-images.githubusercontent.com/49162938/67675022-fc3f2f80-f975-11e9-8133-d1dcb872853d.png)

Best regards!

Reply:
Note also that on most modern Linux distributions `/var/run` is the same as `/run` (more specifically, `/var/run` is a symlink pointing to `/run`), so recommending `/var/run` does not really break anything on Linux (it's actually a bit more portable, because some Linux systems do not actually provide `/run`, but pretty much all of them provide `/var/run`).

Reply:
Change has been added. Thank you for the explanation.